buildSrc/src/main/groovy/org/elasticsearch/hadoop/gradle/fixture/hadoop/HadoopClusterFormationTasks.groovy

```diff
@@ -168,6 +172,16 @@ class HadoopClusterFormationTasks {
                             }
                         }
                     }
+
+                    // Check to see if any of the instance tasks are test cluster aware, and if they are, set the
+                    // es cluster to be whichever cluster was configured, if any
+                    if (instanceInfo.elasticsearchCluster != null) {
```

Comment:
As discussed in person, if this is unnecessary and you can configure it in-place, I think that would avoid a possible point of confusion.